central/cve/converter/utils/convert_utils.go

```diff
@@ -277,6 +288,7 @@ func EmbeddedVulnerabilityToImageCVE(os string, from *storage.EmbeddedVulnerabil
 			CvssV3:       from.GetCvssV3(),
 		},
 		Cvss:         from.GetCvss(),
+		Nvdcvss:      nvdCvss,
```

Comment:
In the protos, `repeated CVSSScore cvss_scores` is added to `ImageCVE` . So that would need to be set here as well.

Reply:
search tag is only added to nvd cvss since we will sort on nvd score. I have not added search tag on cvss metrics https://github.com/stackrox/stackrox/blob/master/proto/storage/cve.proto#L146